ArduCopter/AP_Arming.cpp

```diff
@@ -49,6 +49,9 @@ bool AP_Arming_Copter::run_pre_arm_checks(bool display_failure)
         return mandatory_checks(display_failure);
     }
 
+    // bitwise & ensures all checks are run
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wbitwise-instead-of-logical"
```

Comment:
given how widely we use this, perhaps we should make it global? Or at least put it at the top of the cpp file

Reply:
Done